web/src/lib/utils/file-uploader.ts

```diff
@@ -91,6 +91,9 @@ async function fileUploader(
 		// Set Asset is Favorite to false
 		formData.append('isFavorite', 'false');
 
+		// Set Asset is Archived to false
+		formData.append('isArchived', 'false');
```

Comment:
This does default to false in the database. We should consider making it optional so the mobile app doesn't break while this either (1) isn't implemented there or (2) is released/update before/after the server is.

Reply:
Made `isArchived` optional in `CreateAssetDto`